addons/project/tests/test_project_mail_features.py

```diff
@@ -543,3 +530,31 @@ def test_task_notification_on_user_ids_update(self):
             self.flush_tracking()
         # check that no mail was received for the assignee of the task
         self.assertNotSentEmail(self.user_projectuser.email_formatted)
+
+    def test_task_portal_share_adds_followers(self):
+        """ Test that sharing a task through the portal share wizard adds recipients as followers.
+
+            Test Cases:
+            ===========
+            1) Verify that the portal user is not a follower of the task.
+            2) Create and execute a portal share wizard to share the task with the portal user.
+            3) Verify that the portal user has been added as a follower after sharing.
+        """
+
+        # Verify initial state - our portal user's partner should not be a follower
```

Comment:
the comment is not very useful. The the test case description + the error message are enough to explain the flow of the test.